applications/FluidDynamicsApplication/custom_processes/distance_modification_process.cpp

```diff
@@ -545,6 +672,23 @@ void DistanceModificationProcess::CheckAndStoreVariablesList(const std::vector<s
     }
 }
 
+const std::array<size_t,2> DistanceModificationProcess::GetNodeIDs(
+    const size_t numEdges, 
+    const size_t edgeID) 
+{
+    std::array<std::array<size_t,2>, 3> node_ids_2d {{ {{1,2}}, {{2,0}}, {{0,1}} }};
+    std::array<std::array<size_t,2>, 6> node_ids_3d {{ {{0,1}}, {{1,2}}, {{2,0}}, {{0,3}}, {{1,3}}, {{2,3}} }};
```

Comment:
```suggestion
    std::array<std::array<std::size_t,2>, 3> node_ids_2d {{ {{1,2}}, {{2,0}}, {{0,1}} }};
    std::array<std::array<std::size_t,2>, 6> node_ids_3d {{ {{0,1}}, {{1,2}}, {{2,0}}, {{0,3}}, {{1,3}}, {{2,3}} }};
```

Reply:
Besides, we can move these two outside the function to be allocated once as a `constexpr` in the class types definition.